pkg/security/metrics/metrics.go

```diff
@@ -422,6 +422,13 @@ var (
 	PathResolutionTag = "resolution:path"
 	// AllResolutionsTags is the list of resolution tags
 	AllResolutionsTags = []string{SegmentResolutionTag, ParentResolutionTag, PathResolutionTag}
+
+	// ProcessSourceEvent is assigned to metrics for process cache entries created from events
+	ProcessSourceEventTag = "event"
```

Comment:
I think the tag should be like this "type:event" ? I would create a variable like the other tags, WDYT ?

Reply:
Indeed, fixed.
I guess you mean a variable of type []string to avoid creating the array at runtime ?